src/components/PlantingSites/PlantingSiteCreate.tsx

```diff
@@ -173,7 +173,7 @@ export default function CreatePlantingSite(props: CreatePlantingSiteProps): JSX.
                   </Grid>
                 </Box>
               </Grid>
-              <BoundariesAndPlots plantingSite={record} />
+              {record?.boundary && <BoundariesAndPlots plantingSite={record} />}
```

Comment:
the boundaries and plots component has a rendering for sites without boundaries.. we should clean that up maybe?

Reply:
I'll remove that. thanks!